pkg/mkspipelinerun/list.go

```diff
@@ -0,0 +1,56 @@
+package mkspipelinerun
+
+import (
+	"context"
+	"fmt"
+
+	vr "github.com/MiniTeks/mks-server/pkg/client/clientset/versioned"
+
+	"github.com/spf13/cobra"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog/v2"
+)
+
+var mksPrList = &cobra.Command{
+	Use:   "list",
+	Short: "List PipelineRuns",
+	Annotations: map[string]string{
+		"commandType": "main",
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		list()
+	},
+}
+
+func listcommand() *cobra.Command {
+
+	return mksPrList
+}
+
+func list() {
+	cfg, err := clientcmd.BuildConfigFromFlags("", cfgFile)
+
+	if err != nil {
+		klog.Fatalf("Error building kubeconfig: %v", err)
+	}
+	mksClient, err := vr.NewForConfig(cfg)
+	fet, _ := mksClient.MkscontrollerV1alpha1().MksPipelineRuns(namespace).List(context.TODO(), v1.ListOptions{})
+
+	fmt.Println("Here is List")
+	fmt.Printf("\n")
+	for i := range fet.Items {
```

Comment:
also you can separately create a PRINT func to do the task, for better readability